### DISCUSSION TITLE
Fetch CHANGELOG.md (maven + GitHub) fetches no content

### DISCUSSION THREAD

**Discussion by fjakop** [renovatebot/renovate#13113] [Request Help] [answered]
Hello, we're using renovatebot 31.x on k8s to scan maven projects. Our projects have set <url> and <connection> properties in pom.xml.
When renovate runs in our test setup, it correctly detects the new version of `de.jakop.spielwiese:komponente`

    Wed, Dec 15 2021 8:56:52 am | DEBUG: 1 flattened updates found: de.jakop.spielwiese:komponente (repository=myprivaterepo/de.jakop.spielwiese.myapp)
    Wed, Dec 15 2021 8:56:52 am | DEBUG: Returning 1 branch(es) (repository=myprivaterepo/de.jakop.spielwiese.myapp)
    Wed, Dec 15 2021 8:56:52 am | DEBUG: Fetching changelog: https://github.com/fjakop/de.jakop.spielwiese.komponente (0.0.1 -> 0.0.2) (repository=myprivaterepo/de.jakop.spielwiese.myapp)

and says it would fetch the changelog.

In the repo fjakop/de.jakop.spielwiese.komponente is a CHANGELOG.md containing entries for the tagged versions, but the generated merge request shows only a link to the source code comparison and not the contents of the changelog file.

Is this the intended behaviour?

**Comment by rarkins** [comment, marked as answer]
No, that's not intended. But the cause is this markdown file which doesn't use markdown formatting: https://raw.githubusercontent.com/fjakop/de.jakop.spielwiese.komponente/main/CHANGELOG.md

**Comment by fjakop** [comment]
Oh wow thanks, normally we're usin AsciiDoctor markdown. Is this supported too?

**Reply by rarkins** [reply]
I'm not familiar with that tool. Renovate looks for standard markdown headings like `### foo`

**Comment by fjakop** [comment]
ok thanks a lot, I'll change to markdown.
